packages/venia-ui/lib/components/Wishlist/WishlistButton/wishlistButton.ce.js

```diff
@@ -16,13 +16,13 @@ const ErrorIcon = <Icon src={AlertCircle} attrs={{ width: 18 }} />;
 const WishlistButton = props => {
     const classes = mergeClasses(defaultClasses, props.classes);
 
-    const talonProps = useWishlistButton({ itemOptions: props.itemOptions });
```

Comment:
Is the `useWishlistButton` talon unused now? Can we remove?

Reply:
Yeah, it is, I dint remove the old code yet, once I get approval on this code style, ill go ahead and clean up. (Move files around and delete unnecessary code)